server/src/main/java/org/jetbrains/bsp/bazel/server/sync/ExecuteService.java

```diff
@@ -117,6 +122,12 @@ private Set<BuildTargetIdentifier> selectTargets(java.util.List<BuildTargetIdent
   }
 
   private boolean isBuildable(Module m) {
+    checkManualTag(m);
     return !m.isSynthetic() && !m.tags().contains(Tag.NO_BUILD);
   }
+
+  private WorkspaceContext checkManualTag(Module m) {
```

Comment:
probably name would be better - `isManualTargetBuildable`, also this method should return boolean - it should check the condition - is the manual target buildable (checking is the tag there, and if it is, return value of the flag)

(result of this method is not even used)